src/digitalIdentity-frontend/src/app/pages/home-page/home-page.component.html

```diff
@@ -25,15 +25,15 @@ <h4>
       [ngStyle]="{ cursor: 'pointer' }"
     >
       <div [ngStyle]="{ margin: '50px' }"></div>
-      <mat-icon [ngStyle]="{ color: '500', transform: 'scale(3)' }"
+      <mat-icon class="icon"
         >security</mat-icon
       >
       <div [ngStyle]="{ margin: '50px' }"></div>
```

Comment:
It would be great to remove all of the [ngStyle] properties and add them to the css/scss :)